packages/api/package.json

```diff
@@ -37,6 +37,7 @@
     "tslib": "~1.10.0"
   },
   "devDependencies": {
+    "@babel/parser": "^7.5.5",
```

Comment:
I don't like to add `@babel/parser` to the `api` of Stryker. Stryker core and api are parser agnostic. I would like to keep it that way. Also seem my next remark

Reply:
cool